openverse_catalog/dags/common/storage/media.py

```diff
@@ -36,6 +36,7 @@
 PROVIDER_API = "provider_api"
 
 FILETYPE_EQUIVALENTS = {"jpeg": "jpg", "tif": "tiff"}
+INTEGER_MAXIMUM = 2147483647
```

Comment:
Perhaps a prefix will make it more specific and clear! but I leave it up to you.

```suggestion
PG_INTEGER_MAXIMUM = 2147483647
```

Reply:
Oh that's a great call 😄